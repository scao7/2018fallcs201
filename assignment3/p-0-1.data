537 9948 249 ; 935 1477 348 ; 4609 2295 224 ; 9757 8177 366 ; 9352 3807 497 ; 
4406 2903 200 ; 8658 9605 128 ; 5902 8673 192 ; 1111 7346 366 ; 3139 7183 184 ; 

