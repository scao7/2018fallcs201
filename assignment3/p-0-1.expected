0: 537
1: 9948(537)249
weight: 249
